docs/en/setup/backend/backend-receivers.md

```diff
@@ -128,3 +129,20 @@ receiver-oc:
     gRPCHost: ${SW_OC_RECEIVER_GRPC_HOST:0.0.0.0}
     gRPCPort: ${SW_OC_RECEIVER_GRPC_PORT:55678}
 ```
+
+## Kafka receiver
```

Comment:
The doc on agent side is missing?

Reply:
Read my comment, this is not a receiver.